aspnet/fundamentals/environments.rst

```diff
@@ -88,24 +94,7 @@ ASP.NET Core supports a convention-based approach to configuring an application'
 
 When an ASP.NET Core application starts, the ``Startup`` class is used to bootstrap the application, load its configuration settings, etc. (:doc:`learn more about ASP.NET startup <startup>`). However, if a class exists named ``Startup{EnvironmentName}`` (for example ``StartupDevelopment``), and the ``ASPNETCORE_ENVIRONMENT`` environment variable matches that name, then that ``Startup`` class is used instead. Thus, you could configure ``Startup`` for development, but have a separate ``StartupProduction`` that would be used when the app is run in production. Or vice versa.
```

Comment:
Note that `WebHostBuilder.UseStartup<TStartup>()` overrides detection of environment specific startup types. Environment specific methods still work. See https://github.com/aspnet/Home/issues/1595